### ISSUE TITLE
Error: Exception calling ".ctor" with "1" argument(s): "Could not find a part of the path

### ISSUE THREAD

**Issue by adamrushuk** [zyborg/pester-tests-report#1]
I'm using `v1.3.1` of your GH Action and got the following error:

```powershell
Module [GitHubActions] not found, INSTALLING...
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Installing package 'GitHubActions'                                                 Downloaded 0.00 MB out of 0.01 MB.                                              [                                                                    ]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      Running from [/home/runner/work/_actions/zyborg/pester-tests-report/v1.3.1]

Test Results Path provided as input; skipping Pester tests


VERBOSE: Resolving XML file relative to current directory: /home/runner/work/aks-nexus-velero/aks-nexus-velero/test/pester-test-results.xml
VERBOSE: Resolving default XSL file: /home/runner/work/_actions/zyborg/pester-tests-report/v1.3.1/nunit-report/nunitxml2md.xsl
VERBOSE: Loaded XSL transformer
nunitxml2md.ps1: /home/runner/work/_actions/zyborg/pester-tests-report/v1.3.1/action.ps1:197
Line |
 197 |      & "$PSScriptRoot/nunit-report/nunitxml2md.ps1" -Verbose `
     |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling ".ctor" with "1" argument(s): "Could not
     | find a part of the path
     | '/home/runner/work/aks-nexus-velero/aks-nexus-velero/_TMP/test-results.md'."

##[error]The process '/usr/bin/pwsh' failed with exit code 1
```

I only want to publish the Pester report file, so I've used the following GH Workflow config:

```yaml
    - name: Pester report
      uses: zyborg/pester-tests-report@v1.3.1
      with:
        test_results_path: test/pester-test-results.xml
        report_name: pester_tests
        report_title: Pester Tests
        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Am I missing something?

**Comment by ebekker** [comment]
Thanks, this should be addressed now in the latest [v1.3.2](https://github.com/zyborg/pester-tests-report/releases/tag/v1.3.2).

**Comment by adamrushuk** [comment]
Thanks for fixing so fast!
